### DISCUSSION TITLE
Remove the selected region completely

### DISCUSSION THREAD

**Discussion by laxmangaian** [katspaugh/wavesurfer.js#3549] [Q&A]
Hi, I am trying to remove the selected region completely from the audio. I don't know how to achieve it. Please help me.

**Comment by katspaugh** [comment]
Wavesurfer can't cut or modify your audio in any way, it's just a player.

**Reply by laxmangaian** [reply]
Can you help me to extend it to work like that? Any similar kind of implementation or reference url please

**Reply by katspaugh** [reply]
Try stackoverflow or ChatGPT. Also this thread might help: https://github.com/katspaugh/wavesurfer.js/issues/419
